de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/ShipmentSchedulePA.java

```diff
@@ -1509,7 +1509,7 @@ public void updatePreparationDate_Override(final Timestamp preparationDate, fina
 	@Override
 	public Set<I_M_ShipmentSchedule> retrieveForInvoiceCandidate(@NonNull final I_C_Invoice_Candidate candidate)
 	{
-		final Set<I_M_ShipmentSchedule> schedules;
```

Comment:
keep it final and see below

Reply:
I made it like that to be similar to the one from ReceiptScheduleDAO

Reply:
do the other way around :) final is always better.